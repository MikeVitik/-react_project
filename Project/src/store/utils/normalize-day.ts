export function normalizeDay(currentDate: number) {
  return (currentDate + 6) % 7;
}
