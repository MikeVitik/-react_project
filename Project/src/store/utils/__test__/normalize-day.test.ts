import { normalizeDay } from "../normalize-day";

it("normilaze-day", () => {
  expect([...Array(6)].map((_, day) => normalizeDay(day))).toMatchSnapshot();
});
