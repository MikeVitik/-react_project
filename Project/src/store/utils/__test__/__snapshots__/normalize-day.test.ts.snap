// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`normilaze-day 1`] = `
Array [
  6,
  0,
  1,
  2,
  3,
  4,
]
`;
